v2/tools/generate-techniques-documentation.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
```

Comment:
I would rather have a separate file for YAML vs. documentation

Reply:
Sure, can do. I added to this file to use the `map[stratus.Platform]map[string][]*stratus.AttackTechnique` which contains all information needed for the yaml. But I can extract the `writeYaml` function to a different file. 

Would it be okay to still integrate with this file (as in get the value of the `index` variable and `docs` directory)? 
Or would you rather have a complete separate logic to generate `yaml` files?

Reply:
How about we split in 3 files:
* Processing code to output the map
* Code to generate the docs
* Code to generate the YAML

?
This way we have code reuse and separation of concerns

Reply:
Just added a commit for that, let me know what you think!